go_mod_edits.sh

```diff
@@ -21,7 +21,8 @@
 readonly ABS_PATH="$1"
 
 go mod edit -replace github.com/GoogleCloudPlatform/healthcare-data-harmonization/mapping_engine="${ABS_PATH}"/mapping_engine
-go mod edit -replace github.com/GoogleCloudPlatform/healthcare-data-harmonization/mapping_language/transpiler="${ABS_PATH}"/mapping_language/transpiler
+#go mod edit -replace github.com/GoogleCloudPlatform/healthcare-data-harmonization/mapping_language/transpiler="${ABS_PATH}"/mapping_language/transpiler
```

Comment:
nit: normally we don't comment out code, since you always have the history in the codebase, and you can bring it back anytime.